docroot/themes/humsci/humsci_basic/src/js/shared/prefered-reduced-motion/prefered-reduced-motion.js

```diff
@@ -0,0 +1,31 @@
+(function ($, Drupal, window) {
+  Drupal.behaviors.mySlickBehavior = {
+    // eslint-disable-next-line no-unused-vars
+    attach(context, settings) {
+      // Check for reduced motion preference
+      const mediaQuery = window.matchMedia('(prefers-reduced-motion: reduce)');
+
+      function handleReducedMotionPreference() {
+        const slickInstance = context.querySelector('.slick__slider');
+
+        // Check if the user prefers reduced motion
+        if (!mediaQuery || mediaQuery.matches) {
+          $(slickInstance).on('beforeChange', (event, slick) => {
+            slick.slickSetOption('cssEase', 'none', true);
+          });
+        } else {
+          $(slickInstance).on('beforeChange', (event, slick) => {
+            slick.slickSetOption('cssEase', 'ease', true);
+          });
```

Comment:
While testing more, I think the library already does this by default. Test in a different branch/env.

Reply:
Here's a video https://www.loom.com/share/857ec27d89124493a8f6e14dc3bfe4c4?sid=622fc92f-3bf5-4020-9968-2ea143965b6a

Reply:
Here's how it looks with the fade transition https://www.loom.com/share/60fb89ed8d80476da5dec32de826b5f8?sid=16e771f5-96cb-4796-a1c0-da7c8edfd068